### ISSUE TITLE
The example in docs has wrong package import

### ISSUE THREAD

**Issue by ya3ya6** [necolas/react-native-web#1797]
TL;DR:
  why the example in docs uses `react-native`, not `react-native-web` ? [link to doc](https://github.com/necolas/react-native-web#examples)

Detailed:
  I'm new to react-native-web.
  i first visited the npm page [here](https://www.npmjs.com/package/react-native-web)
  it's linked to this repo, so i came here and i did copy [the example in docs](https://github.com/necolas/react-native-web#examples)
  but it imports normal `react-native` package, not `react-native-web`.
  so i couldn't make it work. however after changing to `react-native-web`, it worked.
  so shouldn't it be : `import { AppRegistry, StyleSheet, Text, View } from 'react-native-web';` ?

**Comment by necolas** [comment]
Please refer to the getting started guide http://necolas.github.io/react-native-web/docs/?path=/docs/overview-getting-started--page
